contrib/kube-prometheus/README.md

```diff
@@ -65,17 +67,17 @@ $ kubectl create -f manifests/
 
 The content of this project consists of a set of [jsonnet](http://jsonnet.org/) files making up a library to be consumed.
 
-Install this library in your own project with [jsonnet-bundler](https://github.com/jsonnet-bundler/jsonnet-bundler#install):
+First create a new project, and install kube-prometheus as a library with [jsonnet-bundler](https://github.com/jsonnet-bundler/jsonnet-bundler#install) (the jsonnet package manager):
```

Comment:
Nitpick - As someone new to jsonnet the wording "create a new project" isn't super clear to me. I like annotated code examples because those are very clear, maybe this can just be changed to comments in the code snippet showing the setup below? Alternately change the wording to something like "create a directory to hold your jsonnet project, initialize it with jsonnet-bundler, then add kube-prometheus as a library in that project" (although this is describing line by line what the code is doing, hence my suggestion to just make it code comments)

Reply:
I like that a lot, thanks for the suggestion!